### DISCUSSION TITLE
Error fetching content: HTTP code 403:  https://anistar.org

### DISCUSSION THREAD

**Discussion by QuAzI** [FreshRSS/FreshRSS#7112] [Q&A]
Neither I try to fetch https://anistar.org or https://anistar.org/rss.xml I always have 403. The same story with Wget. But cURL and Thunderbird scraps feed fine. I tried to play with headers and browser fingerprint but nothing helps.
Any ideas?

**Comment by Frenzie** [comment]
Are you sure about cURL? For me it indeed gives a 403:

```
$ curl https://anistar.org/rss.xml
<!DOCTYPE html>
<!--[if lt IE 7]> <html class="no-js ie6 oldie" lang="en-US"> <![endif]-->
<!--[if IE 7]>    <html class="no-js ie7 oldie" lang="en-US"> <![endif]-->
<!--[if IE 8]>    <html class="no-js ie8 oldie" lang="en-US"> <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en-US"> <!--<![endif]-->
<head>
<title>Attention Required! | Cloudflare</title>
[...]
```

**Reply by QuAzI** [reply]
Strange, now works only with header
```shell
$ curl -H "User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:133.0) Gecko/20100101 Firefox/133.0" https://anistar.org/rss.xml
```
but in FreshRSS agent doesn't help
![image](https://github.com/user-attachments/assets/32ed2889-f49f-4b67-88d4-e0ea9a14ee7a)

**Comment by Alkarex** [comment]
Cloudflare uses a diversity of sniffing method to break the Web, so changing the user agent might not always suffice.
You can also try to go through https://feedburner.google.com or similar

**Reply by QuAzI** [reply]
Thanks, interesting site hint, but...
![image](https://github.com/user-attachments/assets/32db5783-3d4a-42e3-8a48-3db838a39e48)

**Reply by Alkarex** [reply]
Works for me. Makes sure to paste `https://anistar.org/rss.xml`

**Reply by Alkarex** [reply]
No sorry, fails for me as well...

**Reply by Alkarex** [reply]
You can try to reach out to the Web master, showing
https://validator.w3.org/feed/check.cgi?url=https%3A%2F%2Fanistar.org%2Frss.xml

**Reply by QuAzI** [reply]
There is no sense while I can't fetch content. I can't even parse it manually while request blocked
